dev/openstack_kitchensink.yaml

```diff
@@ -0,0 +1,161 @@
+heat_template_version: 2013-05-23
```

Comment:
I don't think this should be added to these directories if we're going to use all the templates in the directory to test with. It won't work on our cloud at all at the moment, so maybe we move these sorts of things to a devstack directory?